src/platform/forms/address.js

```diff
@@ -399,3 +399,223 @@ export function isBlankAddress(address) {
     && isBlank(address.postalCode.value);
 }
 
+export const statesOnlyInPCIU = [
```

Comment:
Is the plan to fetch this information in the future?

Reply:
Yeah, we'd probably fetch the full state and country list. In the meantime, I compared what we fetch in Letters to the existing constants state list and the `statesOnlyInPCIU` is the difference between the two. The countries are totally different in shape and formatting so I just copied those over from the endpoint response.